Root/Preselect.cxx

```diff
@@ -221,16 +221,24 @@ EL::StatusCode Preselect :: execute ()
         wk()->skipEvent();
         return EL::StatusCode::SUCCESS;
       }
+      
+      //if(eventInfo->eventNumber()==22223){
+      //  std::cout << jet->pt() / 1000. << " " << jet->eta() << " " << jet->phi() << " " << jet->m() << " " <<  VD::isSignal(*jet) << std::endl;
+      //}
+    
       if(jet->pt()/1000. < m_jet_minPt)   continue;
       if(jet->pt()/1000. > m_jet_maxPt)   continue;
-      if(jet->m()/1000.  < m_jet_minMass) continue;
-      if(jet->m()/1000.  > m_jet_maxMass) continue;
+      //if(jet->m()/1000.  < m_jet_minMass) continue;
+      //if(jet->m()/1000.  > m_jet_maxMass) continue;
```

Comment:
there should not be a jet mass cut -- due to calibrations, jet masses could be negative.